parsons/__init__.py

```diff
@@ -53,6 +53,7 @@
     ("parsons.databases.mysql.mysql", "MySQL"),
     ("parsons.databases.postgres.postgres", "Postgres"),
     ("parsons.databases.redshift.redshift", "Redshift"),
+    ("parsons.databases.bigquery.bigquery", "BigQuery"),
```

Comment:
I guess I was thinking it would be part of a major release so breaking changes would be expected. But now that I think about that, I don't think there's any reason we can't import it from both places.

Reply:
Moved the file back to the original location and modified the database init file to also import it from there. So both should work

Reply:
Thanks for making this adjustment. Where possible, we want to give folks time to make the change, so in 3-6 we can circle back and remove this entirely.

The docs should signal this deprecation. I missed that this lacks a docs update in my first review, but this should update the docs so that the BigQuery docs are in the Databases folder instead of in the Google folder, both in the git repository and in the docs lefthand sidebar. It probably makes sense to leave a stub in the Google folder, though, that says something like "GoogleBigQuery has been moved to the Databases folder and is available under BigQuery, please change this in your code if you don't want it to break later".

Reply:
I think the big query docs can still live in the same place since I moved the file back to its original place. Unless I'm missing something.

All we've basically done is take the original class and export it in the database package as well

Reply:
I double-checked the docs and I was misremembering how they're structured. There's actually no subfolders at all, including no databases folder. Whoops. We can keep the databases .rst file as is - like how Redshift points to the entry under AWS, BigQuery's spot in the databases listing can keep pointing to Google.

That said, we do still want a note about the rename and the planned deprecation in the docs, so that anyone who checks the docs over the next 3-6 months will be able to proactively rename, and then when we finally do make the breaking change of removing GoogleBigQuery as an option, fewer people will be affected.

Reply:
The more I think about it, I don't think renaming it from GoogleBigQuery to BigQuery which is functionally what it would be to users would be worth a deprecation process. So I just removed the second import path and will change our code.

Hoping to get this merged this week so we can onboard a contractor on Monday!